### PR TITLE
Design implement weekly subscriber

### PR DESCRIPTION
**Pull request by Jaydip-M**
Task : 

1. Design and Implement Weekly subscriber.

### CODE REVIEW

File: src/assets/styles/contest.less [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+.ant-row {
+  display: flex;
+  align-items: center;
+}
+
+.cm-btn,
+.ct-btn {
+  border: 1px solid #b69056;
+  text-transform: uppercase;
```
Comment (divy0124) [inline src/assets/styles/contest.less]: use. utiliy class in this .less

File: src/assets/styles/contest.less [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  background-color: #fff;
+  color: #b69056;
+  &:hover {
+    background: #fffaf1;
+  }
+}
+.ct-btn {
+  background: #b69056;
+  color: #fff !important;
+  &:hover {
```
Comment (divy0124) [inline src/assets/styles/contest.less:29]: use. utiliy class in this .less

File: src/assets/styles/weekly-subscriber.less [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@import './variables.less';
+@import './utility.less';
+
+.ant-btn-icon {
+  color: white;
+}
+.ant-input-search .ant-input-search-button {
+  border-color: #b69056 !important;
+
```
Comment (divy0124) [inline src/assets/styles/weekly-subscriber.less:9]: use. utiliy class in this .less

File: src/index.css [hunk 4]
```diff
@@ -17,3 +17,6 @@ body {
 code {
   font-family: 'Alegreya SC', serif;
 }
+body div .ant-tooltip {
+  display: none;
+}
```
Comment (divy0124) [inline src/index.css]: why this is needed?

File: src/modules/Touchdown/components/Touchdown.js [hunk 5]
```diff
@@ -452,7 +449,10 @@ function Touchdown() {
 
   return (
     <div className="container">
-      <Row className="text-medium font-alegreya mb-20 pointer back-arrow">
+      <Row
+        className="text-medium font-alegreya mb-20 pointer back-arrow"
+        onClick={back}
+      >
```
Comment (divy0124) [inline src/modules/Touchdown/components/Touchdown.js:455]: give proper function name

File: src/modules/Touchdown/components/WeeklySubscriber.js [hunk 6]
```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/order */
+/* eslint-disable import/no-unresolved */
+import { useLazyQuery } from '@apollo/client';
```
Comment (divy0124) [inline src/modules/Touchdown/components/WeeklySubscriber.js:1]: /* eslint-disable no-unused-vars */
/* eslint-disable import/order */
/* eslint-disable import/no-unresolved */

remove this

File: src/modules/Touchdown/components/WeeklySubscriber.js [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+        } else if (search.length === 0) {
+          const { getWeeklySubscribers } = data;
+          const { items, meta } = getWeeklySubscribers;
+          const allItems = [...weeklySubscriber.items, ...items];
+          setWeeklySubscriber({ ...weeklySubscriber, items: allItems, meta });
+        } else {
+          const { getWeeklySubscribers } = data;
+          const { items, meta } = getWeeklySubscribers;
+          setWeeklySubscriber({ ...weeklySubscriber, items, meta });
+        }
```
Comment (divy0124) [inline src/modules/Touchdown/components/WeeklySubscriber.js]: optimisation is requried

File: src/modules/Touchdown/pages/Contest.js [hunk 8]
```diff
@@ -1,7 +1,94 @@
...
+
+    const upcomingWeekStartDate = today
+      .add(1, 'week')
+      .startOf('week')
+      .format(MM_DD_YYYY);
+    const upcomingWeekEndDate = today
+      .add(1, 'week')
+      .endOf('week')
+      .format(MM_DD_YYYY);
+
```
Comment (divy0124) [inline src/modules/Touchdown/pages/Contest.js]: put contest out side of function

File: src/modules/Touchdown/pages/Contest.js [hunk 9]
```diff
@@ -1,7 +1,94 @@
...
+              >
+                weekly subscriber
+              </Button>
+            </Col>
+            <Col>
+              <Button className="cm-btn">weekly leaderboard</Button>
+            </Col>
+            <Col>
+              <Button className="ct-btn" onClick={loadComponent('touchdown')}>
+                create touchdown
```
Comment (divy0124) [inline src/modules/Touchdown/pages/Contest.js:64]: onClick={()=>  setViewComponen('name')}

**Review by divy0124 — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 4 above)
Comment (Jaydip-M) [reply src/index.css]: To remove tooltip on hover when we hover on table col

**Review by divy0124 — APPROVED** [review]

**Pull request merged**